sql/core/src/main/scala/org/apache/spark/sql/execution/SparkStrategies.scala

```diff
@@ -571,9 +571,14 @@ abstract class SparkStrategies extends QueryPlanner[SparkPlan] {
           resultExpressions,
           planLater(child)))
 
-      case PhysicalAggregation(_, _, _, _) =>
+      case PhysicalAggregation(_, aggExpressions, _, _) =>
+        var groupAggPandasUDFNames = Set[String]()
+        aggExpressions.foreach(expr => if (expr.isInstanceOf[PythonUDF]) {
+          groupAggPandasUDFNames += expr.asInstanceOf[PythonUDF].name
+        })
         // If cannot match the two cases above, then it's an error
-        throw QueryCompilationErrors.cannotUseMixtureOfAggFunctionAndGroupAggPandasUDFError()
+        throw QueryCompilationErrors.cannotUseMixtureOfAggFunctionAndGroupAggPandasUDFError(
+          "<" + groupAggPandasUDFNames.mkString(",") + ">")
```

Comment:
Can we use string interpolation here? BTW, is `<...>` format is consistent in the error message?

Reply:
Seems like we usually use single quotes for emphasize the name. Let me change.